deps: Bump express from 4.22.2 to 5.2.1 (#1380)
SAP/ui5-tooling: internal/documentation/docs/pages/extensibility/CustomServerMiddleware.md

# Custom UI5 Server Middleware

> The UI5 Server Extensibility enables you to enhance the functionality of the UI5 Server. You may want to handle requests differently. For example add various headers to a response or parse data of a POST request in a specific way. For this you can plug custom middleware implementations into the internal [Express 5](https://expressjs.com/) server of the UI5 Server module.

Reviewer: ```suggestion
UI5 Server Extensibility lets you extend the UI5 Server. You can handle requests in new ways, for example, by adding headers to a response or parsing POST request data. To do this, you plug custom middleware into the UI5 Server's internal [Express 5](https://expressjs.com/) server.
```

Reviewer 2: Thank you. Applied via https://github.com/UI5/cli/pull/1534

There can be optional configuration parameters which are passed directly to the custom middleware implementation (see below).

> An optional mountPath for which the middleware function is invoked can be provided. It will be passed to the `app.use` call (see [express API reference](https://expressjs.com/en/5x/api.html#app.use)).

Reviewer: ```suggestion
You can also provide an optional `mountPath` to call the middleware function. The system passes this path to the `app.use` call. For more information, see the [express API reference](https://expressjs.com/en/5x/api.html#app.use).
```

Reviewer 2: Thank you. Applied via https://github.com/UI5/cli/pull/1534
